Client/sdk/multiplayer/CMultiplayer.h

```diff
@@ -37,6 +37,14 @@ struct SClothesCacheStats
     uint uiNumRemoved;
 };
 
+enum class EVehicleAimDirection : char
+{
```

Comment:
Hmm, I'm not sure why you changed the prefix to `E`. It's not consistent with the coding style. If you don't like using Hungarian notation, that's completely fine, but it's important to be consistent. I don't think we have any enums with the capital letter "E". I'd rename it to `VehicleAimDirection` or `eVehicleAimDirection`.

Reply:
![image](https://user-images.githubusercontent.com/923242/82157505-c01fdd80-9879-11ea-8dd7-694cbf74ff09.png)

Reply:
Should it be `char` or `unsigned char`, though?

Reply:
Oh my.. 

> Should it be `char` or `unsigned char`, though?

It doesn't make any difference if the range is lower than 128.

Reply:
I looked around and the first enum I found had E. But I changed it now.